analyzers/conversion_overflow.go

```diff
@@ -161,6 +147,87 @@ func isStringToIntConversion(instr *ssa.Convert, dstType string) bool {
 	}
 }
 
+func hasExplicitRangeCheck(instr *ssa.Convert, dstType string) bool {
+	block := instr.Block()
+	dstInt, err := parseIntType(dstType)
+	if err != nil {
+		return false
+	}
+
+	minBoundChecked := false
+	maxBoundChecked := false
```

Comment:
These values don't need to be preinitialized. I suggest just using:

```go
minBoundChecked := checkSourceMinBound(srcInt, dstInt)
maxBoundChecked := checkSourceMaxBound(srcInt, dstInt)
```

Reply:
resolved